scripts/upgrade

```diff
@@ -149,8 +149,6 @@ pushd "$final_path"
 	php7.2 artisan horizon:purge
 popd
 
-
-
 ynh_backup_if_checksum_is_different "$final_path/CONFIG_FILE"
```

Comment:
I think the .env file should be generated again in the upgrade script. An existing application without the __APP_KEY__ won't use it.

Reply:
Done in https://github.com/YunoHost-Apps/pixelfed_ynh/pull/32

Reply:
Update done and build OK